File: UNIT 1/py/kapasitansi.py
# Program menghitung kapasitansi

# Input muatan
muatan = float(input("Masukkan nilai muatan (dalam coulomb): "))

# Input tegangan
tegangan = float(input("Masukkan nilai tegangan (dalam volt): "))

# Hitung kapasitansi
kapasitansi = muatan / tegangan

# Output hasil dengan 2 angka di belakang koma
print("Kapasitansi = {:.2f} Farad".format(kapasitansi))

File: UNIT 1/C/kapasitansi.c
#include <stdio.h>
int main (){
float kapasitansi, tegangan, muatan;
printf("Masukkan nilai muatan (dalam coulomb): ");
scanf("%f", &muatan);
printf("Masukkan nilai tegangan (dalam volt): ");
scanf("%f", &tegangan);
kapasitansi = muatan/tegangan;
printf("Kapasitansi = %.2f Farad\n", kapasitansi);
return 0;
}